### DISCUSSION TITLE
Disable user registration

### DISCUSSION THREAD

**Discussion by lucaberti** [gramps-project/gramps-web#355] [General]
Hello, 
I'm thinking to use Gramps Web as part of our family site, in order to get away from commercial treesites. Since this will be self-hosted, I'm quite worried that unauthorized people will try to create a bunch of users (as usually happens when you leave a WordPress site with registration enabled) just to try to get in. Is there a way to disable registration? Is this a planned feature?

**Comment by DavidMStraub** [comment]
Hi! This is already covered in https://github.com/gramps-project/Gramps.js/issues/338, although a discussion is actually more appropriate :slightly_smiling_face: Feel free to contribute to the existing discussion.

**Comment by lucaberti** [comment]
Sorry: I checked if there where an existing discussion. Not thoughtfully enough...
